src/benchmarklib/benchmark_runner.cpp

```diff
@@ -264,13 +269,19 @@ void BenchmarkRunner::_benchmark_ordered() {
         static_cast<float>(std::chrono::duration_cast<std::chrono::nanoseconds>(_state.benchmark_duration).count());
     const auto duration_seconds = duration_of_all_runs_ns / 1'000'000'000.f;
     const auto items_per_second = static_cast<float>(result.successful_runs.size()) / duration_seconds;
-    const auto num_successful_runs = result.successful_runs.size();
-    const auto duration_per_item =
-        num_successful_runs > 0 ? static_cast<float>(duration_seconds) / static_cast<float>(num_successful_runs) : NAN;
+
+    // Compute mean by using accumulators
+    boost::accumulators::accumulator_set<double, boost::accumulators::stats<boost::accumulators::tag::mean>> acc;
```

Comment:
Neat, I've never seen this.

Could you name the variable `accumulator` please? We try to avoid abbreviations wherever possible.